ui/js/components/perf/compare.js

```diff
@@ -49,8 +49,15 @@ treeherder.component('phCompareTable', {
                 return results;
             }
             return _.filter(results, function(result) {
-                var testCondition = key + ' ' + result.name;
+                var testCondition = key + ' ';
+                if ($attrs.type === 'trend') {
```

Comment:
This could be simplified with an ES6 template literal and ternary operator:

```js
var testCondition = `${key} ${($attrs.type === 'trend') ? result.trendResult.name : result.name}`;
```